src/auth/internal/AuthIntegrationInterface.ts

```diff
@@ -82,13 +82,20 @@ function sendSms(phoneNum: string, featheryClient: any) {
   }
 }
 
-function verifySms(params: {
+function verifySMSOTP(params: {
   fieldVal: string;
   featheryClient: any;
 }): Promise<any> {
   if (authState.authType === 'stytch') return stytchVerifySms(params);
   else if (authState.authType === 'firebase') return firebaseVerifySms(params);
-  else return params.featheryClient.verifySMSOTP(params.fieldVal);
+  else return params.featheryClient.verifyOTP(params.fieldVal, 'sms-otp');
+}
+
+function verifyEmailOTP(params: {
```

Comment:
no need for this additional function here, just call featheryClient.verifyOTP directly